strippy.ps1

```diff
@@ -245,6 +248,21 @@ function log ([String] $Stage, [LEnum] $Type = [LEnum]::Trace, [String] $String,
     }
 }
 
+# Help flag checks
+if ($h -match "true" -or $help -match "true" ) {
```

Comment:
If there was anything I would change it might be this line. While matching the switches to a string will work it would probably be better written as `$h -eq $true -or $help -eq $true` or even just `$h -or $help`

What's written should/will work fine, this is just a matter of syntax.